### PR TITLE
fix(core): mask password in audit log

### PR DESCRIPTION
**Pull request by simeng-li**
## Summary

Mask the password in the audit logs.

TODO:
should encrypt the password ahead before insert into the interaction storage. 

## Testing

test locally
UT added
WX20230216-155417@2x
WX20230216-155237@2x

### CODE REVIEW

**Review by charIeszhao — APPROVED** [review]

**Review by gao-sun — APPROVED** [review]

File: packages/core/src/routes/interaction/index.ts [hunk 1]
```diff
@@ -292,7 +292,7 @@ export default function interactionRoutes<T extends AnonymousRouter>(
       const { event } = interactionStorage;
 
       const log = createLog(`Interaction.${event}.Submit`);
-      log.append({ interaction: interactionStorage });
+      log.append({ interactionStorage });
```
Comment (gao-sun) [inline packages/core/src/routes/interaction/index.ts:295]: should be a separate PR?

Comment (simeng-li) [reply packages/core/src/routes/interaction/index.ts]: removed
